### DISCUSSION TITLE
Server authenticity is not verified message in logs

### DISCUSSION THREAD

**Discussion by madanyang** [plausible/analytics#3583] [Self-Hosted Support]
In the logs I am seeing the following message. Where does this come from? My wild guest is something to do with mail. The real question is how do I fix this as I haven't seen any environment variables addressing this

```
08:00:03.094 [warning] Description: 'Server authenticity is not verified since certificate path validation is not enabled'
Reason: 'The option {verify, verify_peer} and one of the options \'cacertfile\' or \'cacerts\' are required to enable this.'
```

**Comment by ruslandoga** [comment]
👋 @madanyang 

It comes from `:gen_smtp`'s use of SSL. It's not possible to fix right now.

More info: https://github.com/plausible/analytics/discussions/3335#discussioncomment-6958961

**Reply by mlberl** [reply]
Still not working...

**Reply by ruslandoga** [reply]
v2.1.0 would include an alternative SMTP client the would fix this problem.
